packages/nimble-tokens/dist/fonts/css/fonts.css

```
@@ -14,6 +21,14 @@
   font-display: swap;
 }
 
+@font-face {
+  font-family: "Source Sans Pro light-fallback";
```

Are you sure this is actually getting used...? I don't see it referenced anywhere. I'd expect the same naming convention as the other variations of `Source Sans Pro`. ie the name is the same but the font-style and font-weight it targets is different

```suggestion
  font-family: "Source Sans Pro Fallback";
```

Please apply this to the other instances as well and confirm in a manual test they actually leveraged.

---

I had misunderstood how this worked. I have changed the names and tested that they are being used.